src/actuator.c

```diff
@@ -70,12 +70,13 @@ static int momentary_process(cc_actuator_t *actuator, cc_assignment_t *assignmen
 
     // Momentary
     if (assignment->mode & CC_MODE_MOMENTARY)
-    {    
-        if (actuator_value != assignment->value)
+    {   
+        if (actuator->last_value != actuator_value)
```

Comment:
This is not the best way to check if the value is different, because comparing floats directly is unsafe.
See https://github.com/moddevices/cc-slave/blob/fb031deb40ed092803dd2d893945928ae4d645d5/src/actuator.c#L59 for example code right in this file on how to do it better. We basically substract the 2 values and see if the result in absolute is bigger than a certain "delta".